mathesar_ui/src/header/schema-selector/SchemaSelector.scss

```diff
@@ -141,6 +142,17 @@ button.dropdown.trigger.selector {
         }
       }
 
+      .item {
+        > a {
+          > p {
+            margin: 0;
+            padding: 0;
+            overflow: hidden;
+            text-overflow: ellipsis;
```

Comment:
Whitespace wrapping needs to be prevented inorder for text with space to not overflow.

Try adding a schema with spaces. eg., `schema name that has space`. You will see that the ellipsis will not be shown and text will overflow.

An additional css property has to be added: `white-space: nowrap`, to cover that scenario.